packages/ai-bot/main.ts

```diff
@@ -440,9 +446,24 @@ Common issues are:
     }
   });
 
+  let lists: Map<string, MSC3575List> = new Map();
+  lists.set(SLIDING_SYNC_AI_ROOM_LIST_NAME, {
```

Comment:
Do we need to add this list at all if we don't care about the list of rooms?

Reply:
based on what you set here--i think this is a list of rules, not a list of rooms.